src/ImageCarouselReact/widget/components/Carousel.tsx

```diff
@@ -5,116 +5,33 @@
  * This code is under the MIT license found here:
  * https://github.com/react-bootstrap/react-bootstrap/blob/master/LICENSE
  */
-import { IBootstrapProps, prefix } from "../utils/bootstrapUtils";
+import { prefix } from "../utils/bootstrapUtils";
 import ValidComponentChildren from "../utils/ValidComponentChildren";
```

Comment:
Will extract what I'm using and incorporate it into the component code, then discard the rest.